openraft/src/config/config.rs

```diff
@@ -130,7 +130,12 @@ pub struct Config {
     ///
     /// By default, `OpenRaft` purges `applied_log`s from time to time regardless of snapshots, because it assumes once
     /// logs are `applied` to the state machine, logs are persisted on disk. If not so, set this to true.
```

Comment:
> set this to true

maybe "apply this flag" is better.

Reply:
Either would be fine to me. But the explanation should be elaborated. Have a look at my next commit.